File: Python/섹션 2 (완) - 코드 구현력 기르기/4. 대표값 (중상)/AA.py
import sys
#sys.stdin=open('input.txt', 'rt')

n=input()
arr=list(map(int,input().split()))

sum=0
for e in arr:
    sum+=e
avg=sum/len(arr)
avg=round(avg)

resNum=0
min=float('inf')
maxValue=0
for i in range(len(arr)):
    if min > abs(avg-arr[i]):
        min = abs(avg-arr[i])
        resNum = i
        maxVal = arr[i]

    elif min == abs(avg-arr[i]):
        if maxVal < arr[i]: # 점수가 더 높은 학생
            min = abs(avg-arr[i])
            resNum = i
            maxVal = arr[i]
        elif  maxVal == arr[i]:
            if resNum > i: # 순서가 더 빠른 학생
                min = abs(avg-arr[i])
                resNum = i
                maxVal = arr[i]

# print(arr[resNum], resNum+1)
print(avg, resNum+1)

File: Javascript/6. 홀수/AA.js
// A, B, C 세 수만을 입력 받는 문제
function solution(arr){
    let sum=0
    
    let min=arr[0]
    for(let i=0;i<arr.length;i++){
        if(arr[i]%2==1){
            sum+=arr[i]
            
            if(min<arr[i]){
                min=arr[i]
            }
        }
    }
    console.log(sum)
    console.log(typeof(sum))
    return min
}
str="12 77 38 41 53 92 85"
arr=str.split(",")